evalai/get_starter.py

```diff
@@ -0,0 +1,13 @@
+import click
+import subprocess
+
+from evalai.utils.config import STARTER_URL
+
+
+@click.group(invoke_without_command=True)
+def get_starter():
+    """
+    Download the EvalAI-Starter
+    """
+    download_path = click.prompt("Enter directory to store EvalAI-Starters in", type=str, hide_input=False)
```

Comment:
@takitsuse if you are using `EvalAI-Starter` in the above comment use the same here or change both to `EvalAI-Starters`. It's to make the messages consistent in the project